Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -496,6 +496,9 @@ public void FilterData()
         /// <param name="value">New cell value.</param>
         public void UpdateEditCell( string fieldName, object value )
         {
+            if ( editState == DataGridEditState.None )

```

Comment:
What's the behaviour if the State is New for example? Will both Update and Read work alright?

Reply:
It's the same. In both states, the same dictionary is used.